NEWS

```diff
@@ -1,3 +1,14 @@
+master
+------
+
+Enhancements
+ - Complete contact properties saving between profiles
```

Comment:
-------------------------------^ there 
and ----------------------------------------------------------^ there 
:trollface: 

Probably something like "More contact's fields are saved" will be better. The "saving between profiles" thing is a way (that probably only developer will understand) to refer that something is stored in sogo's database.